e2e/app/feeoraclev2.go

```diff
@@ -0,0 +1,45 @@
+package app
+
+import (
+	"context"
+
+	"github.com/omni-network/omni/lib/contracts/feeoraclev2"
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/log"
+)
+
+func DeployFeeOracleV2(ctx context.Context, def Definition) error {
```

Comment:
```suggestion
// DeployFeeOracleV2 deploys fee oracle v2.
func DeployFeeOracleV2(ctx context.Context, def Definition) error {
```

add godocs to exported funcs